versions.properties

```diff
@@ -5,7 +5,7 @@
 ## Generated by $ ./gradlew refreshVersions
 ## Please, don't put extra comments in that file yet, keeping them is not supported yet.
 
-plugin.android=4.1.0-rc02
+plugin.android=4.1.0
```

Comment:
Or even 4.1.1 if you want to upgrade that one :)
I guess it was AS bugging you to upgrade to allow Gradle sync… 😅

Reply:
Yes, exactly 😊 I will update to 4.1.1